elyra/pipeline/tests/test_processor_kfp.py

```diff
@@ -192,11 +193,10 @@ def test_collect_envs(processor):
     assert 'USER_NO_VALUE' not in envs
 
 
-@pytest.mark.skip(reason="failing - need to evaluate function at kfp compiler level")
-def test_processing_url_runtime_specific_component(monkeypatch, processor, sample_metadata):
+def test_processing_url_runtime_specific_component(monkeypatch, processor, sample_metadata, tmpdir):
     # Assign test resource location
-    url = 'https://raw.githubusercontent.com/elyra-ai/elyra/master/elyra/' \
-          'pipeline/resources/kfp/filter_text_using_shell_and_grep/component.yaml'
+    url = 'https://raw.githubusercontent.com/elyra-ai/elyra/master/' \
+          'elyra/pipeline/tests/resources/components/filter_text.yaml'
```

Comment:
I guess this test won't succeed until it's merged.  Have you run a test that (temporarily) points to a different branch?  (I really don't want to have to introduce env overrides for branches, so am okay waiting and fixing if broken.)

Reply:
Yes, I tested against the resource in my branch (`url = https://raw.githubusercontent.com/kiersten-stokes/elyra/server-test/elyra/pipeline/tests/resources/components/filter_text.yaml`) and it's passing!